src/api/helpers/map-CIS-countries/map-CIS-country/map-esra-classification/index.ts

```diff
@@ -0,0 +1,27 @@
+import { EXTERNAL_API_DEFINITIONS, EXTERNAL_API_MAPPINGS } from '../../../../constants';
+
+const { CIS } = EXTERNAL_API_DEFINITIONS;
+
+/**
+ * mapEsraClassification
+ * Transform a country's ESRAClassificationDesc field into a consistent string
+ * @param {String} Risk category
+ * @returns {String} Consistent ESRA classification
+ */
+const mapEsraClassification = (str: string): string | null => {
+  if (str === CIS.ESRA_CLASSIFICATION.STANDARD) {
+    return EXTERNAL_API_MAPPINGS.CIS.ESRA_CLASSIFICATION.STANDARD;
+  }
```

Comment:
I don't think we need to - if the value changes (in an external system), this mapping would fail either way, unless it's updated to the exact same constant.

It is very unlikely that ESRA classifications will be re-worded.